include/tvTypes.h

```diff
@@ -371,6 +371,13 @@ typedef enum tvPQParameterIndex {
     PQ_PARAM_CMS_LUMA_YELLOW,                    //!< Picture parmaeter is Component luma yellow
     PQ_PARAM_CMS_LUMA_CYAN,                     //!< Picture parmaeter is Component luma cyan
     PQ_PARAM_CMS_LUMA_MAGENTA,                  //!< Picture parmaeter is Component luma magenta
+    PQ_PARAM_WB_RED_GAIN,                       //!< Picture parmaeter is Component Red Gain
```

Comment:
could this follow current naming convention, like,
PQ_PARAM_<xyz>_<abc>_RED

Reply:
Done